runtime/node/src/shared/composerBot.ts

```diff
@@ -16,6 +25,7 @@ import { SimpleCredentialProvider, SkillValidation } from 'botframework-connecto
 import { BotSettings } from './settings';
 import { SkillConversationIdFactory } from './skillConversationIdFactory';
 import { getSettings, getProjectRoot, getRootDialog } from './helpers';
+// import { CustomActionComponentRegistration } from '../customaction/customActionComponentRegistration';

```

Comment:
Consider adding inline explanation of what this line is for and when to enable it